spec/models/element_spec.rb

```diff
@@ -33,6 +33,10 @@ module Alchemy
     end
 
     describe '.definitions' do
+      it "should allow erb generated elements" do
+        expect(Element.definitions.collect {|el| el['name']}).to include('erb_element')
```

Comment:
Prefer `map` over `collect`.<br>Space between { and | missing.<br>Space missing inside }.